lib/ts/recipe/session/sessionFunctions.ts

```diff
@@ -22,6 +22,9 @@ import { Helpers, JWKCacheMaxAgeInMs } from "./recipeImplementation";
 import { maxVersion } from "../../utils";
 import { logDebugMessage } from "../../logger";
 
+
+// TODO: fix type error `TS2367: This condition will always return 'false' since the types '(userContext: any) => Promise<"VIA_TOKEN" | "VIA_CUSTOM_HEADER" | "NONE">' and 'string' have no overlap.`
+
```

Comment:
in this file we need to eliminate use of helpers.config.antiCsrf by passing in it's values to the function directly. If we do this, then do we need to have config obj in the helpers at all?